frontend/common/src/helpers/dateUtils.ts

```diff
@@ -58,27 +58,32 @@ export function getDateRange({
       );
 }
 
+// parameters for the relativeExpiryDate function
+export type formatDateOptions = {
+  date: Date;
+  formatString: string;
+  intl: IntlShape;
+  timeZone?: string;
+};
```

Comment:
Pure nitpick, but can we use `PascalCase` for type names. Also, I think the comment needs updating:

`// parameters for the formatDate function`

Reply:
Good stuff!  Gotta keep things tidy.  [Tidy naming and constants](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4407/commits/676bd3492b6353dd42529568b685aaa426028e92)